css/selectors/grouping.json

```diff
@@ -0,0 +1,55 @@
+{
+  "css": {
+    "selectors": {
+      "type": {
+        "__compat": {
+          "description": "Grouping selectors (<code>,</code>)",
+          "mdn_url": "https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Grouping_selectors",
```

Comment:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Selector_list",
```

Reply:
See: https://github.com/mdn/browser-compat-data/pull/532/files

The link should refer to the `Groping selectors` page.

Reply:
Done.